kubernetes/job.yaml

```diff
@@ -14,7 +15,7 @@ spec:
     spec:
       containers:
         - name: descheduler
-          image: descheduler:latest
+          image: docker.io/aveshagarwal/descheduler:0.9.0 # TODO: change to official gcr.io registry when ready
```

Comment:
Same here. Should just image name `descheduler` be used here?

Reply:
See comment https://github.com/kubernetes-sigs/descheduler/pull/208#discussion_r370973435